native/src/components/PoiDetails.tsx

```diff
@@ -93,19 +64,24 @@ const PoiDetails = ({ poi, language, distance }: PoiDetailsProps): ReactElement
       <HorizontalLine />
       <AddressInfo location={poi.location} language={language} />
       <HorizontalLine />
-      {!!(website || phoneNumber || email) && (
-        <>
-          {contactInformationCollapsibleItem}
-          <HorizontalLine />
-        </>
-      )}
+      {contacts.length > 0 &&
+        contacts.map(contact => (
+          <Contact
+            key={contact.headline ?? contact.website ?? contact.name ?? contact.phoneNumber}
+            headline={contact.headline}
```

Comment:
❓ is there a reason why we pass here all the contact data attributes separately.
We could just pass the `contact : ContactModel` and spread it in the component